harmonica/forward/point_mass.py

```diff
@@ -178,9 +178,14 @@ def point_mass_gravity(
     cast = np.broadcast(*coordinates[:3])
     result = np.zeros(cast.size, dtype=dtype)
     # Prepare arrays to be passed to the jitted functions
-    coordinates = (np.atleast_1d(i).ravel().astype(dtype) for i in coordinates[:3])
-    points = (np.atleast_1d(i).ravel().astype(dtype) for i in points[:3])
+    coordinates = tuple(np.atleast_1d(i).ravel().astype(dtype) for i in coordinates[:3])
+    points = tuple(np.atleast_1d(i).ravel().astype(dtype) for i in points[:3])
     masses = np.atleast_1d(masses).astype(dtype).ravel()
+    # Sanity checks
+    if masses.size != points[0].size:
+        raise ValueError(
+            "Masses array must have the same size as number of point masses."
```

Comment:
It would be good to include the values in the error message so we know what's wrong. This message could be triggered deep in a function call stack and the user might have no idea what is going on.

Reply:
Done! Let me know if you like it or I should change anything. Besides, I've opened #104 in order to improve the tesseroids error message for invalid density array.